docs/source/broadcasting.md

```diff
@@ -8,6 +8,8 @@ on port 1935. You can broadcast into Livepeer using this port.
 
 The following instructions assume that you have followed the [installation](installation.html) and [quickstart](quickstart.html) instructions.
 
+Some of the `livepeer` commands in these instructions require an ETH node JSON-RPC URL to be provided via the `-ethUrl <ETH_RPC_URL>` flag. See the [quickstart](quickstart.html) instructions for more details on obtaining an ETH node JSON-RPC URL.
```

Comment:
Considering we changed "ETH" to "Ethereum" in the go-livepeer PR, could we do that here as well? Sorry for the annoying change request.

Reply:
Fixed in 43ab625 and ecebcd7